lib/worker-pipeline.js

```diff
@@ -23,6 +35,13 @@ module.exports = async function ({ targets }) {
         throw error
       }
     }
+
+    // Depending on how the default export is performed, and on how the code is
+    // transpiled, we may find cases of two nested "default" objects.
+    // See https://github.com/pinojs/pino/issues/1243#issuecomment-982774762
+    if (typeof fn === 'object') fn = fn.default
+    if (typeof fn === 'object') fn = fn.default
```

Comment:
duplicated code by mistake?

Reply:
No, it's very strange and ugly, but it covers a real problem with some transpiled files 😢 .